chef/cookbooks/neutron/attributes/default.rb

```diff
@@ -31,6 +31,10 @@
 default[:neutron][:network][:gre_start] = 1
 default[:neutron][:network][:gre_stop] = 1000
 
+default[:neutron][:vxlan][:vni_start] = 4096
+default[:neutron][:vxlan][:vni_stop] = 99999
+default[:neutron][:vxlan][:multicast_group] = "239.1.1.1"
+
```

Comment:
Why don't you use the same naming schema as for gre? like ```default[:neutron][:network][:vxlan_vni_start]```. I think your version is nicer but inconsistent with the other code. Isn't it?